proto/celestia/blob/v1/tx.proto

```diff
@@ -15,6 +18,8 @@ service Msg {
 
 // MsgPayForBlobs pays for the inclusion of a blob in the block.
 message MsgPayForBlobs {
+  option (cosmos.msg.v1.signer) = "signer";
```

Comment:
[question]

where can I find out what this does?

Reply:
https://docs.cosmos.network/main/build/building-modules/protobuf-annotations#signer